### ISSUE TITLE
Can we push a Hugo 0.76.5 extended image?

### ISSUE THREAD

**Issue by sdirix** [eclipse-cbi/dockerfiles#26]
Hi!

We use the popular [Syna theme](https://github.com/okkur/syna) for Hugo. Sadly the theme is incompatible with hugo `0.78.1` and hugo `0.81.0` which are the two versions which are currently offered by [`eclipsecbi/hugo_extended`](https://hub.docker.com/r/eclipsecbi/hugo_extended/tags).

Instead we require Hugo 0.76.5.
We experience issues for quite a while, see for example these dropdowns not working: https://www.eclipse.org/ecp/emfforms/downloads/.
This is fixed with a downgrade to Hugo 0.76.5.
We also tried updating the theme and using the latest version of Hugo but sadly either doesn't work.

Can we build and push an image based on `hugo 0.76.5`? If yes, how do you want to go about it? For example we could open a PR for the Jenkinsfile and revert immediately after merging?

**Comment by mbarbero** [comment]
sure, go ahead, create a PR

**Comment by fredg02** [comment]
Since no PR was opened for this, I'm closing this issue after almost two years.

**Comment by sdirix** [comment]
Actually I did open a [PR](https://github.com/eclipse-cbi/dockerfiles/pull/15) and it was merged, however neither me nor someone else updated the issue. Sorry for that.

**Comment by sdirix** [comment]
@fredg02 You're right, if I remember correctly we managed to live with the later Hugo version. Thanks for closing.
